### PR TITLE
Trying to fix #6 and make it prettier

### LINKED ISSUE #6: Clean up table generation
Two issues:
- Code has been copy-pasted and can be turned into a for loop
- There are two separate functions that return table items, and they behave slightly differently.

https://github.com/ianklatzco/twitter-to-bsky/blob/892a27dc3af8ac79f311320cf0c287f4f0bdd261/webapp/main.py#L241

### PR DESCRIPTION
**Pull request by Heartade**
Managed to result in 43 lines fewer code??

### CODE REVIEW

**Comment by Heartade** [comment]
Also #9

**Comment by Heartade** [comment]
I guess eventually we'd need to turn to Django or something that provides easier front-end development.

File: webapp/main.py [hunk 1]
```diff
@@ -129,53 +183,59 @@ def process_json(input_json=None):
         # entities -> url -> urls
 
         desc_urls = entities.get('description').get('urls')
-        if not desc_urls: continue 
+        if not desc_urls or len(desc_urls) == 0:
+            continue
```
Comment (ianklatzco) [inline webapp/main.py:187]: at first was worried len(e) in the second half of the `or` might throw an exception but looks like it's fine
```
>>> e = None
>>> if not e or len(e) == 0:
...     print("foo")
...
foo
```

File: webapp/main.py [hunk 2]
```diff
@@ -129,53 +183,59 @@ def process_json(input_json=None):
...
+            "http://", "").replace("https://", "")
         print(bsky_username)
 
         # bsky_username = get_bsky_username(description) # from bio
 
         print('got here')
-        session = atprototools.Session(os.environ.get("BSKY_USERNAME"), os.environ.get("BSKY_PASSWORD"))
-        bsky_did = session.resolveHandle(bsky_username).json().get('did')
-        profile_json = session.getProfile(bsky_did).json()
+        # TODO I get 'atprototools' has no attribute 'Session' error. Perhaps it's a new feature and not yet live on PIP
```
Comment (ianklatzco) [inline webapp/main.py:213]: @Heartade Do `pip install --upgrade atprototools`

**Pull request merged**